### PR TITLE
bump TensorKit compat

### PR DESCRIPTION
**Pull request by lkdvos**

### CODE REVIEW

**Comment by Jutho** [comment]
Thanks @lkdvos . Maybe you could use this PR to also update the CI configuration of TensorKitManifolds.jl , so that it tests 1.6, latest 1 stable, and nightly? And maybe check with formatting etc?

**Comment by Jutho** [comment]
This looks ready? Can I merge?

**Comment by lkdvos** [comment]
Yes, should be everything.

File: src/TensorKitManifolds.jl [hunk 1]
```diff
@@ -28,50 +28,51 @@ checkbase(x, y, z, args...) = checkbase(checkbase(x, y), z, args...)
 # the machine epsilon for the elements of an object X, name inspired from eltype
 eleps(X) = eps(real(eltype(X)))
```
Comment (lkdvos) [inline src/TensorKitManifolds.jl:29]: Should we make this scaleps? Should we move this to TensorKit?

Comment (Jutho) [reply src/TensorKitManifolds.jl]: Perhaps, but I would prefer `scalareps`. Which is also just much better than `eleps`.

File: Project.toml [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+version = "0.6.1"
 
 [deps]
 LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 TensorKit = "07d1fe3e-3e46-537d-9eac-e9e13d0d4cec"
 
 [compat]
-TensorKit = "0.6,0.7,0.8,0.9,1"
+TensorKit = "0.11"
 julia = "1"
```
Comment (Jutho) [inline Project.toml:12]: julia 1.6 here in compat?

File: src/auxiliary.jl [hunk 3]
```diff
@@ -72,23 +72,23 @@ function _subtractone!(a::AbstractMatrix)
 end
 function _one!(A::AbstractMatrix)
```
Comment (Jutho) [inline src/auxiliary.jl:73]: There is an equivalent `_one!` in TensorKit that we can probably recycle.

**Pull request merged**